README.md

```diff
@@ -12,11 +11,10 @@ Either as an input or generates as an output.
 
 ## Available schemas
 
-- `TEP.json`: Description of [Target Enabling Packages](https://platform-docs.opentargets.org/target/chemical-probes-and-teps#target-enabling-packages).
-- `chemical_probes.json`: Description of the [chemical probes](https://platform-docs.opentargets.org/target/chemical-probes-and-teps#chemical-probes) data model.
-- `disease_target_evidence.json`: Description of disease-target evidence input. Separate schemas are defined for each data sources.
+- `TEP.json`: description of [Target Enabling Packages](https://platform-docs.opentargets.org/target/chemical-probes-and-teps#target-enabling-packages).
+- `chemical_probes.json`: description of the [chemical probes](https://platform-docs.opentargets.org/target/chemical-probes-and-teps#chemical-probes) data model.
+- `disease_target_evidence.json`: description of disease-target evidence input. Separate schemas are defined for each data sources.
 - `target_safety.json`: Description of [target safety](https://platform-docs.opentargets.org/target/safety).
-- `molecular_interactions.json`: Definition of the [molecular interactions](https://platform-docs.opentargets.org/target/molecular-interactions) input.
+- `molecular_interactions.json`: definition of the [molecular interactions](https://platform-docs.opentargets.org/target/molecular-interactions) input.
 - `molecular_interactions-output.json`: Definition for the molecular interactions dataset generated by the Platform ETL.
-
-##
+- `gene-essentiality.json`: definition of gene essentiality target annotation. Developed based on [DepMap](https://depmap.org/) CRISPR/CAS9 screens.
```

Comment:
```suggestion
- `gene-essentiality.json`: definition of gene essentiality target annotation. Developed based on [DepMap](https://depmap.org/) CRISPR/CAS9 screens.

## Dev environment set up
```bash
# Clone the project
git clone https://github.com/opentargets/json_schema
cd json_schema

# Install Poetry
curl -sSL https://install.python-poetry.org | python -

# Install the project dependencies
poetry install
poetry shell
```
```